DOCS-2336 and DOCS-2369 Synthetics CI/CD Testing Edits (#11447)
DataDog/documentation: content/en/synthetics/api_tests/dns_tests.md

## Overview

DNS tests allow you to proactively monitor the resolvability and lookup times of your DNS records using any nameserver. If resolution is unexpectedly slow or a DNS server answers with unexpected A, AAAA, CNAME, TXT, or MX entries, Datadog sends you an alert with details on the failure, allowing you to quickly pinpoint the root cause of the issue and fix it.

DNS tests can run from [managed][1] and [private locations][2], allowing you to run tests on public domains as well as internal services that might be using internal DNS.

## Permissions

By default, only users with the [Datadog Admin and Datadog Standard roles][3] can create, edit, and delete Synthetic DNS tests. To get create, edit, and delete access to Synthetic DNS tests, upgrade your user to one of these two [default roles][3].

If you have access to the [custom role feature][4], add your user to a custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.

## Configuration

> Now that you have chosen to create a [`DNS` test][5], define your test's request.

Reviewer: ```suggestion
After choosing to create a [`DNS` test][5], define your test's request.
```
